api/firmware/btc.go

```diff
@@ -240,20 +243,64 @@ func (device *Device) BTCSign(
 	if err != nil {
 		return nil, err
 	}
+
+	isInputsPass2 := false
 	for {
 		switch next.Type {
 		case messages.BTCSignNextResponse_INPUT:
 			inputIndex := next.Index
+			input := *tx.Inputs[inputIndex].Input
+
+			var hostNonce []byte
+			if supportsAntiklepto && isInputsPass2 {
+				hostNonce = bytesOrPanic(32)
```

Comment:
in Go this would typically be called `mustRandom`.

but apart from the name nit, this is another place where panic is not ideal. returning an error in cases like this would result in a much better UX overall. it is possible for crypto/rand.Read to fail.

Reply:
Done